bp_common/test/src/demos/src/accel_dp.c

```diff
@@ -0,0 +1,82 @@
+#include <stdint.h>
```

Comment:
Can we condense to 1 .c test by using #defines  ?

Reply:
1 test per type that is.

Reply:
Good idea! Will do that.

Reply:
Good idea! Will do that.